.github/workflows/pushdockerimage.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2023 Klepsydra Technologies AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Docker KPSR Tutorial
+
+on:
+  push:
+    branches: [ main ]
+
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  docker:
+    strategy:
```

Comment:
Strategy is not required. We don't have matrix here.

Reply:
Indeed you're right, I forgot to delete these lines.
Done in 7a4c95b.